### DISCUSSION TITLE
ARTS ORGANIZATION -  Want potential audience members with disabilities to be aware which festival performances have which accessibility measures

### DISCUSSION THREAD

**Discussion by mariel-marshall** [pair-cg/general-contributions#29] [User Stories]
### Original User Story: 
“As a festival organizer I want to have standard language for accessibility measures across the sector so that audiences know what to expect and can search based on accessibility measure type (i.e. relaxed performance, audio described, ASL interpreted). When the performing arts community has standard language or "style guide" for how to refer to these performances as well as a set of criteria for what they represent”

# 1. Need:
-Get potential audience members with disabilities to be aware which festival performances have which accessibility measures

# 2. Action:
_What is an action that can be taken towards fulfilling that need?_

- Inform potential audience members (including people with disabilities) about the festival and it's productions, e.g.,  include information about accessibility, relaxed performance, audio described, ASL interpreted)

           - post/share information about the festival productions on our festival website, including information about accessibility 
               - post/share information in accessible formats about the festival productions on our festival website, including information about accessibility
           - post/share information about the festival productions on social media
           - post/share information about the festival productions on listing websites     
           - share information about the festival productions with media for coverage 

### Information to be communicated 
**Current access measure information the festival is interested in communicating to potential patrons** 

key information [ticketing information, venue information] communicated about attending the festival:

1. Format: Video
- Language(s) of video: ASL
- Closed captioning
- Audio description 
- Voice-over

2. Format: Text [video transcripts]
- Language: English
- screen-reader friendly [what does this mean for the text to be screen reader friendly?]

3. Access Measures
- Closed captioning
-   Auto-Transcribed Captioning
-   Live-Transcribed Captioning
- Audio description
- screen-reader friendly [what does this mean for the text to be screen reader friendly?]
- Interpretation (ASL, LSQ, Deaf, Swiss-German Sign Language (DSGS))
- Interpreters available for booking (ASL) 
- Relaxed performances
      - alternative to traditional relaxed performances with quiet spaces and closed circuit feed for performances.
- Assisted Performances [please describe what this is?]
- Assisted Performances Supplement [please describe what this is?] - a text document which gives information to patrons about accessibility measures available for specific performances

4. Personnel:
- Accessibility Manager - a new position created by the festival to ...[please describe the main responsibilities of this role]

5. Venue Information
- Site-specific venues are required to be physically accessible
- Level performance space of stage offered 
- Accessible audience entrance is the same as the general audience entrance (previously there had been a separate accessible entrance, which meant patrons using a mobility device had to enter through a different location). 
- lobby cleaning station for patron's with mobility devices

6. Accessibility Standard Requirements
- established mandatory minimum standards of accessibility:
     -Captions for videos
     -Audio accompanied by full transcripts
     -Alternative descriptions for all images
     -Screen-reader friendly text documents

7. Additional details
- Welcomed our first Deaf participant into TENT 2020, providing full ASL coverage for the program.
- Welcomed our first Deaf-led community partner, Deaf Spirit Theatre, as participants in the Fringe Collective.
- Hosted an online Audio Described Listening Party of Fringe Collective offerings with Blind host, Christine Malec, and audio describer, Kat Germain, in response to wants voiced by Blind and partially sighted community.
- Collaborated with Phoenix the fire and Teklawatheta, both new Deaf-led BIPOC service providers, to present a Deaf Interpreted Kidsfest Story Hour during our Fringe Collective live streamed programming series in July.
- Implemented self-description protocol [can you define or describe what this is?] for Fringe staff and presenters for live and recorded events.
- Accessible Program Guide supplement for the Fringe Collective.
- Visual Venue Guides
- Venue Accessibility Information (guide)
- Compiling a list of universally accessible venues for potential site-specific participants to use when applying to the Toronto Fringe Festival.
- Refining the use of our language around what an ‘accessible’ venue means, recognizing we have been talking more specifically about patrons, rather than artists and technicians where there are still many barriers in many theatres.
- Creating a set of functional illustrated ASL cards with a local Deaf artist to better enable our non-signing staff to communicate with Deaf patrons.
- LSQ performances (Quebec Sign Language)

# 3. Desired Outcome:
_What is the desired outcome?_

Target audiences are reached

# 4. Perspective:
_From what perspective is this user story coming from?_
Arts Administrator

# 5. Stakeholder Group:
_Which stakeholder group most closely fits the perspective of this User Story?_

- Arts Organization
- Audiences
- patrons/artists/funders/volunteers/sponsors

# 6. Additional info:
_Here's more information._

TO Fringe Access Measures: https://fringetoronto.com/accessibility/access-measures

**Comment by birk** [comment]
@mariel-marshall Does it make sense to specify the scope of the audience? Is it local? While physical accessibility seems to be quite universal, I have no idea how accessible ASL is for francophones. And whether something is conceived as relaxed or not, can only be said in regard to a specific audience, I think.

**Reply by mariel-marshall** [reply]
@birk What do you mean by scope of audience? Can you explain more. 

There seems to be a need to communicate that the show is Interpreted, or offers interpretation and _then_ a way to specify which languages of interpretation are available e.g. ASL, LSQ, Deaf, Swiss-German Sign Language (DSGS) etc. 

# Physical Accessibility 
Physical accessibility is a whole area requiring much more work. What does physically accessible mean? What does it mean for the venue, for seating protocols, for parking etc. Physically accessible seems to have a lot of different meanings from place to place. In Ontario the AODA defines minimum requirements for accessible buildings here, which is likely different than in Switzerland: http://www.ohrc.on.ca/en/moving-towards-barrier-free-services-final-report-restaurant-accessibility-initiative/accessibility-ontarians-disabilities-act-and-building-code

Communicating how physically accessibile the actual performance/experience is a different thing entirely 
- For example, if the show requires audience participants get up on stage at any point, but the stage is not accessible to someone with a mobility device, the performance is not an accessible experience for audiences, even if the building or seating areas are.

# Relaxed Performance (rp)
This also requires some thought and collaboration with experts in this area. In Canada, RP is used to convey a specific set of features. However, different festivals have different implementations of it. 

The best resource I have found for it is https://bodiesintranslation.ca/relaxed-performance-exploring-accessibility-in-the-canadian-theatre-landscape/: 

Relaxed Performance (RP): What is a relaxed performance?
A Relaxed Performance (RP) is a performance in which there are modifications to the theatre environment and sometimes the performance itself. The question of how to ensure that arts patrons can access arts spaces—and how to open up arts spaces to those who have never considered themselves “arts patrons”—is a critical question for our time. RP contends with this question. RP aims to open up the theatre space to welcome differences. Rather than requiring people to stay seated and listen silently, RPs invite attendees to move, speak, leave and return, eat, etc. They also often include other modifications:
• Dimming house lights
• Reducing sound levels
• Reducing or removing strobing lightsand flashes
• Providing a “chill-out space”
• Creating a visual story (gives people information about what to expect at a venue and in a performance)
• Making introductions to the cast before the performance

Another definition: 
Standard relaxed performances offer an introduction at the start of the show and have a relaxed attitude to noise and movement, including reducing extreme sound and lighting effects, leaving house lights on at a low level, and allowing patrons to come and go.

[Theatre Passe Muraille Relaxed Performance:](https://www.passemuraille.ca/relaxed-performances/) Relaxed Performances are designed to welcome audience members and their families who could benefit from a more relaxed environment, including those on the autism spectrum. The set-up includes: a pre-show introduction to the actors/characters, less intense sound and lighting effects, a very low level house light in the audience, the ability for anyone to come and go from the space, and a calm space outside the theatre. 

The Melbourne Fringe uses the term "Sensory Friendly Performance": These are performances where steps have been taken towards making the event or specific performances more sensory friendly, but they may not have all elements expected of a Relaxed Performance. For example, they may make changes to the lighting and sound to make it more sensory friendly but not have a break out area to chill out during the event. Check the ‘Accessibility’ section on each event page for dates of sensory friendly performances. https://melbournefringe.com.au/festival-info/access-information/

They have a few other interesting categorizations as well: 

https://melbournefringe.com.au/festival-info/access-information/

Language no Barrier - This symbol indicates that audiences can understand and engage with the event without a comprehensive understanding of the English language.

Aural Ratings - Our aural ratings are designed to give audience members who are blind or low vision a better understanding of how they can access an event.
- Aural Rating 100% – The event is entirely sound-based with no visual component (e.g. a sound art installation or a show performed in complete darkness)
- Aural Rating 75% – The event is primarily sound-based, with visuals only incidental to the work
- Aural rating 50% – The event has both sound and visual components, but sight isn’t essential to engage properly with the work

Tactile Tours
- Free audio described tactile tours may be available for some events.
 
Content Warnings
All events have been asked to provide warnings, including whether an event contains sudden loud noises, strobe lighting or potentially triggering content. These will be advertised on each event’s online listing. Please read them carefully and get in touch if you have any queries.

**Reply by birk** [reply]
@mariel-marshall I just was wondering to which degree such information can have the same meaning to the members of a very generic audience or whether one has to define a more specific audience to make certain information meaningful. So, I wasn't aware that relaxed performance defines primarily certain adjustments in the overall setup but e.g. doesn't refer to the content of a show and things like trigger warnings. The latter, I guess, would require an understanding of what kind of audience is addressed.

**Comment by mariel-marshall** [comment]
Great resource: https://bodiesintranslation.ca/relaxed-performance-exploring-accessibility-in-the-canadian-theatre-landscape/
